mlflow/ml-package-versions.yml

```diff
@@ -242,6 +242,17 @@ spacy:
     run: |
       pytest tests/spacy/test_spacy_model_export.py --large
 
+spark:
```

Comment:
If necessary, we could (temporarily) create a new github actions workflow to run tests.